Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "rustls-ffi"
-version = "0.9.2"
+version = "0.10.0-pre.1"
```

Comment:
Would you please bump this version to `0.10.0`? Thanks!

Reply:
Good catch, thanks! Fixed in [4fc4331](https://github.com/rustls/rustls-ffi/pull/311/commits/4fc433160ad778b679cf95abbd6eb94c81eba227)